addons/mail/static/src/core/common/message_model.js

```diff
@@ -74,6 +74,14 @@ export class Message extends Record {
             return this.id < this.thread.lastSelfMessageSeenByEveryone.id;
         },
     });
+    isReadBySelf = Record.attr(false, {
+        compute() {
+            return (
+                this.thread.selfMember?.seen_message_id?.eq(this) &&
+                this.thread.selfMember?.new_message_separator >= this.id
```

Comment:
Shouldn't it be `>`? As `new_message_separator` is placed before the message.

Reply:
This field is used to determine if we have to send a request to the server, it's the case if the server value is either greater or the same than this message. So I think it is correct.

Reply:
I'm confused, from what I understand if the `new_message_separator` is the same value as the message id, it means the separator is before the message, so the message is not read.

Reply:
Oh right, it makes sense. I tested it as well and it works as expected.